tests/api/test_BitMask.py

```diff
@@ -140,11 +140,22 @@ def test_equal(self):
         self.assertEqual(mask_a, mask_b)
 
     def test_subset_a_list(self):
-        list_ = [4, 7, 9, 1]
-        mask = BitMask.from_bin("0b0101")
-        target_masked_list = [7, 1]
-        masked_list = mask.masked_select_from_list(list_)
-        self.assertEqual(target_masked_list, masked_list)
+        n = 1000
+        list_ = [randint(0, 1) for _ in range(n - 2)] + [0, 1]
+        mask = BitMask.from_length(n)
+        for index, item_ in enumerate(list_):
+            if item_ == 0:
+                mask.unset_kth_bit(index)
+            else:
+                mask.set_kth_bit(index)
+
+        all_ones = mask.masked_select_from_list(list_)
+        mask.x = mask.x ^ (2 ** n - 1) # inverse
```

Comment:
please use our inverse function we have on the BitMask itself

Reply:
![grafik](https://user-images.githubusercontent.com/65946090/126487154-3c347845-d959-420f-8ecc-bb3998b4684e.png)


It's not implemented yet... But I'll implement it and use it - good call.

Reply:
I thought we already had the magic happening but great call to just do it :)